Command/SystemInfoDumpCommand.php

```diff
@@ -8,25 +8,55 @@
  */
 namespace EzSystems\EzSupportToolsBundle\Command;
 
+use EzSystems\EzSupportToolsBundle\SystemInfo\Collector\SystemInfoCollector;
```

Comment:
Uh, that's somewhat unclear, but I agree that `support_tools.system_info.collector_registry` is better than `support_tools.info_collector.registry`.

Having `support_tools.system_info` before other parts is good. Both `collector_registry` and `collector.registry` works, `registry_collector` not so much.